javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -513,48 +513,46 @@ public static ClientMethodMapper getInstance() {
                     }
 
                     if (settings.isContextClientMethodParameter()) {
-                        JavaVisibility visibility = methodVisibility(ClientMethodType.SimpleAsync, true);
-                        if (visibility != NOT_GENERATE) {
-                            addClientMethodWithContext(methods,
-                                    builder.methodVisibility(visibility),
-                                    parameters);
-                        }
+                        addClientMethodWithContext(methods,
+                                builder.methodVisibility(methodVisibility(ClientMethodType.SimpleAsync, true)),
+                                parameters);
```

Comment:
`settings.isLowLevelClient()` in outer if statement can be removed.

Reply:
good catch. removed.